web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -49,6 +53,9 @@ const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
     firing: true,
     pending: true,
     inactive: true,
+    unknown: false,
+    err: false,
+    ok: false,
```

Comment:
Just so I understand correctly, you are introducing three new rule states here, and we have show/hide buttons for the original inactive/pending/firing, but not for the new ones, and it seems the new ones are always shown, with no option of filtering them. I also don't know if we should be adding three new buttons, but curious what your thinking is here.